Code/Tools/FBuild/FBuild/Main.cpp

```diff
@@ -106,6 +106,9 @@ int Main( int argc, char * argv[] )
         VERIFY( SetPriorityClass( GetCurrentProcess(), BELOW_NORMAL_PRIORITY_CLASS ) );
     #endif
 
+    // Prevent sleep when fbuild is running.
+    Thread::SetContinuousThreadExecutionMode();
```

Comment:
Will this also prevent Windows from sleeping if `FBuildWorker` is running in the tray?

Reply:
Ah this isn't shared with the worker 👍